report/renderer/performance-category-renderer.js

```diff
@@ -311,8 +311,9 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
       ({acronym: 'All'}),
       ...filterableMetrics,
     ]);
+    const nowTs = Date.now();
     for (const metric of filterChoices) {
-      const elemId = `metric-${metric.acronym}`;
+      const elemId = `metric-${metric.acronym}-${nowTs}`;
```

Comment:
FWIW we have a utility for this over in [`Util.getUniqueSuffix`](https://github.com/GoogleChrome/lighthouse/blob/d3504385b8c91a89e6cb6fe590c075dba92adb0c/report/renderer/util.js#L518-L526), used for unique IDs for SVG references in pwa-category-renderer: https://github.com/GoogleChrome/lighthouse/blob/d3504385b8c91a89e6cb6fe590c075dba92adb0c/report/renderer/pwa-category-renderer.js#L167-L172